src/main.py

```diff
@@ -58,8 +58,17 @@ def standby():
 
     Once launch is detected, this method will return.
     """
-    # TODO: Implement this state
-    pass
+  # TODO: Implement this state
+    gpio = GPIOReader(False)
+    RetrieveData = gpio.retrieveData()
+
+    lsm = RetrieveData.__readLSM9DS1()
+
+    while abs((lsm.acceleration_x + lsm.acceleration_y + lsm.acceleration_z)/3) < 10:
```

Comment:
I think we should discus this detection.

It would probably be more accurate to get total acceleration by getting the length of the vector:

```suggestion
    while np.sqrt(np.dot(lsm.acceleration, lsm.acceleration)) < LAUNCH_ACCELERATION_THRESHOLD:
```

This is assuming that you import numpy as np
```python
import numpy as np
```

Could be cleaned up further into:
```python
    while vec_len(lsm.acceleration) < 10.0:
```
And then define some global function elsewhere in the code:
```python
def vec_len(v):
  np.sqrt(np.dot(v, v))
```

**p.s.** Implied in the solution is that `lsm.acceleration` should return a List of x, y, and z. This should be added at some point.

Reply:
Also `LAUNCH_ACCELERATION_THRESHOLD` should be a global constant that we can use everywhere.

Makes it easy to see at the top of the file and change if necessary.